### PR TITLE
CIV-14112 CCD Event: Discontinue this claim

### PR DESCRIPTION
**Pull request by German-A-Amoros**
### JIRA link (if applicable) ###
**[CIV-14112](https://tools.hmcts.net/jira/browse/CIV-14112)**

### Change description ###
CCD Event: Discontinue this claim

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java [hunk 1]
```diff
@@ -97,8 +98,6 @@
 import static uk.gov.hmcts.reform.civil.callback.CaseEvent.RESUBMIT_CLAIM;
 import static uk.gov.hmcts.reform.civil.callback.CaseEvent.SERVICE_REQUEST_RECEIVED;
 import static uk.gov.hmcts.reform.civil.callback.CaseEvent.SET_ASIDE_JUDGMENT;
-import static uk.gov.hmcts.reform.civil.callback.CaseEvent.SETTLE_CLAIM;
```
Comment (hmcts-version1-ignacio) [inline src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java:100]: Why are these removed?

**Review by ShwetaTandel-hmcts — APPROVED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by Omaira-Melo-Hmcts — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]
